### ISSUE TITLE
Filter field in cloudflare_notification_policy terraform resource is not documented

### ISSUE THREAD

**Issue by eze-kiel** [cloudflare/terraform-provider-cloudflare#1406]
### Confirmation

- [X] My issue isn't already found on the issue tracker.
- [X] I have replicated my issue using the latest version of the provider and it is still present.

### Terraform and Cloudflare provider version

```
Terraform v1.0.10
on linux_amd64
+ provider registry.terraform.io/cloudflare/cloudflare v3.4.0
```

### Affected resource(s)

* `clickhouse_alert_fw_ent_anomaly`
* `clickhouse_alert_fw_anomaly`
* `origin_error_rate_alert`

### Terraform configuration files

```hcl
resource "cloudflare_notification_policy" "clickhouse_alert_fw_ent_anomaly" {
  account_id = var.cloudflare_account_id
  name = "Advanced Security Events Alert"
  description = "Cloudflare detects a spike in security events against one of our zones"
  enabled     =  true
  alert_type  = "clickhouse_alert_fw_ent_anomaly"

  webhooks_integration {
    id   =  cloudflare_notification_policy_webhooks.env.id
  }
}

resource "cloudflare_notification_policy" "clickhouse_alert_fw_anomaly" {
  account_id = var.cloudflare_account_id
  name = "Security Events Alert"
  description = "Cloudflare detects a spike in security events against one of our zones"
  enabled     =  true
  alert_type  = "clickhouse_alert_fw_anomaly"

  webhooks_integration {
    id   =  cloudflare_notification_policy_webhooks.env.id
  }
}
```

### Expected output

A successful output or more explanations in the provider doc about how to use the filters.

### Actual output

```
╷
│ Error: error creating policy Origin Error Rate Alert: HTTP status 400: Filters selection must be provided to create a policy. (17103)
│ 
│   with cloudflare_notification_policy.origin_error_rate_alert,
│   on notifications-policies.tf line 37, in resource "cloudflare_notification_policy" "origin_error_rate_alert":
│   37: resource "cloudflare_notification_policy" "origin_error_rate_alert" {
│ 
╵
╷
│ Error: error creating policy Security Events Alert: HTTP status 400: Filters selection must be provided to create a policy. (17103)
│ 
│   with cloudflare_notification_policy.clickhouse_alert_fw_anomaly,
│   on notifications-policies.tf line 121, in resource "cloudflare_notification_policy" "clickhouse_alert_fw_anomaly":
│  121: resource "cloudflare_notification_policy" "clickhouse_alert_fw_anomaly" {
│ 
╵
╷
│ Error: error creating policy Advanced Security Events Alert: HTTP status 400: Filters selection must be provided to create a policy. (17103)
│ 
│   with cloudflare_notification_policy.clickhouse_alert_fw_ent_anomaly,
│   on notifications-policies.tf line 133, in resource "cloudflare_notification_policy" "clickhouse_alert_fw_ent_anomaly":
│  133: resource "cloudflare_notification_policy" "clickhouse_alert_fw_ent_anomaly" {
│ 
╵
```

### Steps to reproduce

1. create a notification webhook
2. create some notification policies without filters
3. try to apply them

### Additional factoids

The incomplete doc: https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs/resources/notification_policy#filters

**Comment by eze-kiel** [comment]
I have checked the documentation (of both Cloudflare notifications and the terraform provider) and I can't find which values this field is supposed to have.

**Comment by zestysoft** [comment]
I just ran into this myself -- I don't think the provider even enumerates this list itself - you have to get it from cloudflare:

https://api.cloudflare.com/#notification-alert-types-get-alert-types

**Comment by eze-kiel** [comment]
Yeah I did this to have the notification listing, but there is nothing about `filters` in the API response :(

**Comment by cheonu** [comment]
facing the same issue myself. Referencing the [cloudflare developer documentation](https://api.cloudflare.com/#notification-policies-create-notification-policy) does not help either. It would be nice to have more details in the terraform documentation

**Comment by revathyrams** [comment]
Apologies for the delay in response!

For these 3 alert types 
* `clickhouse_alert_fw_ent_anomaly`
* `clickhouse_alert_fw_anomaly`
* `origin_error_rate_alert`

you need a mandatory `filters` selection when you create a notification policy. I've included examples for each filter here.

For `origin_error_rate_alert`, an slo and a list of zones.
```
 "filters": {
                "slo": ["99.9"], // valid slos are `99.7`, `99.8` and `99.9`
                "zones": [
                    "my-zone" // a list of active zones you want to be alerted on.
                ]
            }
```
For `clickhouse_alert_fw_anomaly`, a list of zones you want to be alerted on.
```
"filters": {"zones": []}
```

For `clickhouse_alert_fw_ent_anomaly` a list of services and zones.
```
"filters": {
                "services": ["services"],
                "zones": ["my-zone"]
            }
```

I hope this unblocks you for now. In the meantime we'll work on updating our documentation to reflect this.

**Comment by eze-kiel** [comment]
Hello @revathyrams, thanks for the answer!

I have a question regarding `clickhouse_alert_fw_ent_anomaly`:

```
"filters": {
                "services": ["services"],
                "zones": ["my-zone"]
            }
```

What are the different values supported in the service list? I suppose there is a specific format for each one

Thanks!

**Comment by revathyrams** [comment]
you can also find them under `source` as listed here - https://developers.cloudflare.com/logs/reference/log-fields/zone/firewall_events/

**Comment by eze-kiel** [comment]
thanks!

**Comment by jacobbednarz** [comment]
see https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs/resources/notification_policy for documentation on this resource.

**Comment by rikribbers** [comment]
> In the meantime we'll work on updating our documentation to reflect this.

Where can I find this updated documentation?
